recipes/compas-RV2/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "compas-RV2" %}
+{% set version = "1.4.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ec515356bf1f6f3e01aabca1ead4fd6baf26bee264a5963dbeb6a688ad9e70fa
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --ignore-installed --no-cache-dir -vvv
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - compas >=1.13.3
+    - compas_cloud >=0.3.0
+    - compas_tna >=0.1.5
+    - compas_skeleton >=1.1.4
+
+test:
+  imports:
+    - compas_rv2
```

Comment:
```suggestion
test:
  imports:
    - compas_rv2
  requires:
    - pip
  commands:
    - pip check
```

Reply:
Since you are sourcing all of your packages from PyPI, please add `pip check` to the test section of all your packages.